### ISSUE TITLE
EVSSM deblurring prior (L_pr) seems missing / not applied in current training code

### ISSUE THREAD

**Issue by songwenhao123** [PotatoBigRoom/CoherentGS#4]
Hi authors, thanks for releasing the code and paper.

I’m trying to reproduce the paper results and noticed that the EVSSM deblurring prior loss L_pr (the photorealistic/deblurring prior) does not seem to be applied during training in the current codebase.

Is L_pr implemented in the public code? If yes, could you point to the exact file/function and how it is enabled?
Are there any required flags / configs / checkpoints needed to activate L_pr?
If L_pr is not included, do you plan to release it or can you share how it should be integrated?
I’d appreciate any guidance to correctly reproduce the reported metrics.

Thanks!

**Comment by PotatoBigRoom** [comment]
感谢你的关注，我已经更新了我的代码，希望这对你有帮助，主要流程如下。
`if cfg.ref_perceptual_weight > 0 and ref_image is not None:
                colors_mid_nchw = colors_mid.permute(0, 3, 1, 2)
                ref_image_nchw = ref_image.unsqueeze(0).permute(0, 3, 1, 2)
                if ref_image_nchw.shape[-2:] != colors_mid_nchw.shape[-2:]:
                    ref_image_nchw = F.interpolate(
                        ref_image_nchw,
                        size=colors_mid_nchw.shape[-2:],
                        mode="bilinear",
                        align_corners=False,
                    )
                ref_perc_loss = self.perceptual_loss(
                    torch.clamp(colors_mid_nchw, 0.0, 1.0),
                    torch.clamp(ref_image_nchw, 0.0, 1.0),
                )
                loss = loss + cfg.ref_perceptual_weight * ref_perc_loss`

**Comment by pranav-asthana** [comment]
Line 1765 of `simple_deblur_difix.py` has an undefined variable: `colors_mid`. Can you please check and clarify how to initialize this. Thanks!
